src/message_filter.rs

```diff
@@ -15,6 +15,7 @@
 // Please review the Licences for the specific language governing permissions and limitations
 // relating to use of the SAFE Network Software.
 
+#![allow(deprecated)]
```

Comment:
When we disable lints, we normally try and keep the affected scope as small as possible (unless that means duplicating the macro a silly number of times or it is needed for a majority of the functions in a given file), and it's good to add a todo if it's really meant as a temporary fix.